test/test_moduleLoad.py

```diff
@@ -34,3 +34,14 @@ def test__dbConfig(self):
         for namespace in list(dbConfig.get_ns_list()):
             self.assertEqual(dbConfig.get_dbid('PFC_WD_DB', namespace), 5)
             self.assertEqual(dbConfig.get_dbid('APPL_DB', namespace), 0)
+
+def test_BlockUseSwsssdk():
+    # change working folder to import swsssdk in subprocess
+    swsssdk_path = os.path.join(modules_path, 'src')
+    wd = os.getcwd()
+    os.chdir(swsssdk_path)
+
+    result = subprocess.run(["python", "-c", "import swsssdk;exit()"], capture_output=True)
+    assert "deprecated" in result.stderr.decode("utf-8")
```

Comment:
>assert [](http://example.com/codeflow?start=4&length=6)

if assert False, the next statement will be skipped, and there is side effect.

Reply:
Fixed